pom.xml

```diff
@@ -28,7 +28,7 @@
         <opentelemetry-instrumentation.version>2.5.0</opentelemetry-instrumentation.version>
         <opentelemetry-semconv>1.25.0-alpha</opentelemetry-semconv>
 
-        <jenkins.version>2.440.3</jenkins.version>
+        <jenkins.version>2.414.3</jenkins.version>
```

Comment:
we can do https://github.com/jenkinsci/junit-sql-storage-plugin/pull/417 instead

Reply:
I saw that the Jenkins OTel Plugin stil has a lot of users on 2.414.x so downgrading makes sense to me. I'll defer to you.